django_project/api/osm_api_client.py

```diff
@@ -0,0 +1,263 @@
+                raise ElementDeletedApiError(
+                    response.status_code,
+                    response.reason,
+                    payload
+                )
+            raise ApiError(response.status_code, response.reason, payload)
+        if return_value and not response.content:
+            raise ResponseEmptyApiError(
+                response.status_code,
+                response.reason,
+                ''
+            )
+
+        if self._debug:
+            error_msg = (
+                '%s %s %s'
+                % (time.strftime('%Y-%m-%d %H:%M:%S'), method, path)
+            )
+            print(error_msg)
+        return response.content
+
+    def create_node(self, data, comment=None):
+        """Create OSM node data and push it to OSM instance through OSM api.
+
+        :param data: OSM Node data.
+        :type data: dict
+            example: {
+                'lat': latitude of node,
+                'lon': longitude of node,
+                'tag': {},
+            }
+
+        :param comment: Changeset comment.
+        :type comment: str
+
+        :return: OSM changeset data.
+        :rtype: dict
+            example: {
+                'id': id of node,
+                'lat': latitude of node,
+                'lon': longitude of node,
+                'tag': dict of tags,
+                'changeset': id of changeset of last change,
+                'version': version number of node,
+                'user': username of last change,
+                'uid': id of user of last change,
+                'visible': True|False
+            }
+        """
+        changeset_tags = {}
+        if comment:
+            changeset_tags.update({
+                'comment': comment
+            })
+
+        self.ChangesetCreate(changeset_tags)
+        changeset = self.NodeCreate(data)
+        self.ChangesetClose()
+
+        return changeset
+
+    def update_node(self, data, comment=None):
+        """Update OSM node data and push it to OSM instance through OSM api.
+
+        :param data: OSM Node data.
+        :type data: dict
+            example: {
+                'id': id of node,
+                'lat': latitude of node,
+                'lon': longitude of node,
+                'tag': {},
+                'version': version number of node,
+            }
+
+        :param comment: Changeset comment.
+        :type comment: str
+
+        :return: OSM changeset data.
+        :rtype: dict
+            example: {
+                'id': id of node,
+                'lat': latitude of node,
+                'lon': longitude of node,
+                'tag': dict of tags,
+                'changeset': id of changeset of last change,
+                'version': version number of node,
+                'user': username of last change,
+                'uid': id of user of last change,
+                'visible': True|False
+            }
+        """
+        changeset_tags = {}
```

Comment:
i see that some function has same procedural but has just 1different line
`changeset = self.NodeUpdate(data)`, `changeset = self.NodeCreate(data)`, etc
could you create main function that calls same method, bust just calls that different function for each method?